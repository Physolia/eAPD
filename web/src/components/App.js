import React from 'react';

import Routes from '../pages/Route';
import Wrapper from './Wrapper';
import Broadcast from './Broadcast';
import AriaAnnounce from '../containers/AriaAnnounce';
import AutoScrollToElement from './AutoScrollToElement';

const App = () => (
  <Wrapper>
    <AriaAnnounce />
    <Broadcast />
    <AutoScrollToElement />
    <Routes />
  </Wrapper>
);

export default App;
