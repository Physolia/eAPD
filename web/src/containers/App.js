import React from 'react';

import Routes from '../pages/Route';
import Wrapper from '../components/Wrapper';
import Broadcast from '../components/Broadcast';
import AriaAnnounce from '../components/AriaAnnounce';
import AutoScrollToElement from '../components/AutoScrollToElement';

const App = () => (
  <Wrapper>
    <AriaAnnounce />
    <Broadcast />
    <AutoScrollToElement />
    <Routes />
  </Wrapper>
);

export default App;
