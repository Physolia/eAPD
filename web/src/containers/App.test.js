import { shallow } from 'enzyme';
import React from 'react';

import App from './App';

describe('App component', () => {
  test('renders correctly', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
