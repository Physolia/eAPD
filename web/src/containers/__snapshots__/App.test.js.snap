// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<withRouter(Wrapper)>
  <Connect(AriaAnnounce) />
  <Broadcast />
  <Connect(AutoScrollToElement) />
  <Routes />
</withRouter(Wrapper)>
`;
